File: openapi/code_samples/JavaScript/jobs/get.js
const axios = require('axios')

await axios.get("https://api.joby.ai/v0/jobs", {
  params: {
    and: ['node', 'react'],
    location: ['san francisco', 'bay area'],
    days: 30,
    limit: 5
  },
  headers: {
    'x-api-key': 'YOUR_API_KEY'
  }
})

File: openapi/code_samples/Python/jobs/get.py
import requests

requests.get(
    url="https://api.joby.ai/v0/jobs",
    params={
        "and": ["python", "django"],
        "location": ["remote", "new york"],
        "salaryFrom": 80000,
        "limit": 5,
    },
    headers={
        "x-api-key": "YOUR_API_KEY",
    },
)
